demisto_sdk/validation/secrets.py

```diff
@@ -332,10 +332,17 @@ def get_generic_white_list(whitelist_path):
     @staticmethod
     def get_packs_white_list(whitelist_path):
         final_white_list = []
+        files_white_list = []
+
         if os.path.isfile(whitelist_path):
             with io.open(whitelist_path, mode="r", encoding="utf-8") as secrets_white_list_file:
-                final_white_list = secrets_white_list_file.read().split('\n')
-        return final_white_list, [], []
+                temp_white_list = secrets_white_list_file.read().split('\n')
+            for i in range(len(temp_white_list)):
+                if '/' in temp_white_list[i]:
```

Comment:
doesn't seem like a good identifier for files, try to think of something else.
Like os.path.exists